modules/core/src/main/scala/org/tesselation/http/routes/ClusterRoutes.scala

```diff
@@ -38,13 +36,26 @@ final case class ClusterRoutes[F[_]: Async](
             case PeerIdInUse(id) => Conflict(s"Peer id=${id} already in use.")
             case PeerHostPortInUse(host, port) =>
               Conflict(s"Peer host=${host.toString} port=${port.value} already in use.")
-            case SessionAlreadyExists =>
-              Conflict(s"Session already exists.")
+            case SessionDoesNotExist =>
+              Conflict(s"Session does not exist.")
```

Comment:
Why does it result in a 409 error? Isn't this an expected behavior when joining, that the session will not exist?

Reply:
No, when you try to join to the node that does not have a session, you shouldn't be able to join.

Reply:
@tbekas I rephrased the error message to be more verbose

Reply:
you were right.

Reply:
Cool, thanks for checking it.